website/sidebars.json

```diff
@@ -0,0 +1,20 @@
+{
+  "docs": {
+    "Introduction": [
+      "introduction/quick-start",
```

Comment:
We'll need to create some actual Markdown content, and replace these lines with the IDs from those files.

Reply:
Where do i get this content from, `src/README.md` for now?

Reply:
Yeah, that's what I suggested in my earlier comments.